app/domain/authentication/util/check_authenticator_secret_exists.rb

```diff
@@ -0,0 +1,40 @@
+require 'command_class'
+require 'conjur/fetch_required_secrets'
+
+module Authentication
+  module Util
+
+    # Command class that returns boolean value indicating if resource exists
+    CheckAuthenticatorSecretExists = CommandClass.new(
+      dependencies: {
+        resource_class: ::Resource,
+        logger: Rails.logger
+      },
+      inputs: %i[conjur_account authenticator_name service_id var_name]
```

Comment:
why we do no send the `authentication_parameters` ?

Reply:
because authentication_parameters is an objectclass for jwt  and we want this class be good for other authenticators too